run_model-dep-full_limits.sh

```diff
@@ -7,97 +7,89 @@ MODEL=$3
 ANALYSISTYPE=$4
 
 if [[ $ANALYSISTYPE == "classic" ]]; then
-    analysis="mssm_vs_sm_classic"
+    analysis="bsm-model-dep-full"
+    sm_like_hists="sm125"
+    replace_with_sm125=1
+    categorization="classic"
     if [[ $TAG == "auto" ]]; then
         TAG="cmb_classic"
     fi
 else
-    analysis="mssm_vs_sm_h125"
+    analysis="bsm-model-dep-full"
+    sm_like_hists="sm125"
+    replace_with_sm125=1
+    categorization="with-sm-ml"
     if [[ $TAG == "auto" ]]; then
-        TAG="cmb_h125_fixed"
+        TAG="cmb_with_ml"
     fi
 fi
 # Szenarios from here: https://twiki.cern.ch/twiki/bin/view/LHCPhysics/LHCHWGMSSMNeutral?redirectedfrom=LHCPhysics.LHCHXSWGMSSMNeutral#Baseline_scenarios
 ### MSSM scenarios #####
 if [[ $MODEL == "mh125" ]]; then
     wsoutput="ws_mh125.root"
     modelfile="13,Run2017,mh125_13.root"
-    min_mass=110
```

Comment:
Why is this not necessary anymore?

Reply:
Ok, I see it is fixed to the same available values for every scenario now.

Reply:
the parameters `minTemplateMass` and `maxTemplateMass`  are there for available **templates** we have for signals, not for the mass values in the scenarios.

So in our case, this should now be always 60 and 3500

Reply:
Yes, absolutely makes sense. I just saw that it was deleted and only realized later that is was added again at another place of the code.